docs/docs/development/hardware-integration/pointers.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Pointing Devices
+sidebar_label: Pointers
+---
```

Comment:
I took a stab at this. I decided not to create any "shared docs" bit, since the specifics/flow was just subtly different enough it would have led to a more awkward end result in the docs. LMK what you think.